examples/component-definition/xml/example-component.xml

```diff
@@ -30,7 +30,7 @@
          </implemented-requirement>
       </control-implementation>
       <control-implementation uuid="22dbff65-9729-449f-9dfc-4e5fee0906de"
-                              source="../../../content/fedramp.gov/xml/FedRAMP_HIGH-baseline_profile.xml">
```

Comment:
This file is generated by CI/CD. Any change to it will have no effect, since it will be replaced with a newly generated version. Can you revert this file?

Reply:
Woops, I seem to have let that slip through, apologies, will revert!

Reply:
OK fixed it up, ready for review again, @david-waltermire-nist!